apps/explorer/src/components/footer/footerLinks.ts

```diff
@@ -1,65 +1,42 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
-type FooterItem = {
-    category: string;
-    items: { title: string; href: string }[];
-};
+type FooterItem = { title: string; href: string }[];
+
 export type FooterItems = FooterItem[];
 export const footerLinks = [
```

Comment:
seems like we can delete this file with the new footer?

Reply:
deleted